### PR TITLE
Added untested advance concrete mixer

### PR DESCRIPTION
**Pull request by Eclipse-04**
added a bigger 3x3 concrete mixer
(untested, i can't play mindustry anymore due to openGL function problems)
also a larger Uranium Refinery

### CODE REVIEW

**Comment by Eclipse-04** [comment]
Get your **f\*cking @ss** over here

**Comment by BHLuotianyi** [comment]
Remind to add the change to [bundles](https://github.com/THCworkshopCN/THC-Industry/tree/dev/bundles)

**Comment by BHLuotianyi** [comment]
You don't need to add `""` for strings in hjson

File: content/blocks/crafting/adv-concrete-mixer.hjson [hunk 1]
```diff
@@ -0,0 +1,35 @@
+type: LiquidConverter
+name: Concrete mixer
+//you can balace stuffs if you want to
+description: "Mix sand and iron together to make concrete, require large amount of extra water"
+health: 1250
```
Comment (BHLuotianyi) [inline content/blocks/crafting/adv-concrete-mixer.hjson:5]: A factroy shouldn't be so _TANKED_. Maybe the HP should be around 850.

File: content/blocks/crafting/adv-concrete-mixer.hjson [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+size: 3
+requirements:
+[
+  refined-iron/50
+  copper/65
+  junior-circuitboard/45
```
Comment (BHLuotianyi) [inline content/blocks/crafting/adv-concrete-mixer.hjson:11]: Maybe we shouldn't use circuitboards as a kind of main _contruction_ material.It should be more similar to a _tech limiter (e.g:the factory can be only built after you start producing high-rank circuitboards) _ or a _time cosumer (I mean time to develop)_
Maybe we can use more basic materials,and use less of this (or maybe,one middle-rank board)

Comment (BHLuotianyi) [reply content/blocks/crafting/adv-concrete-mixer.hjson:11]: Also,for it's a _senior_ factory,concrete should be used.
Concrete should be a material that has to be produced in huge amounts.(Like silicon,or lead,or something else)

File: content/blocks/crafting/adv-concrete-mixer.hjson [hunk 3]
```diff
@@ -0,0 +1,35 @@
+type: LiquidConverter
+name: Concrete mixer
+//you can balace stuffs if you want to
+description: "Mix sand and iron together to make concrete, require large amount of extra water"
+health: 1250
+size: 3
+requirements:
+[
+  refined-iron/50
+  copper/65
```
Comment (BHLuotianyi) [inline content/blocks/crafting/adv-concrete-mixer.hjson:10]: A 3*3 factory should use more basic materials.My ideas is that building these _"High-rank-junior-things"_ should be a small chanllege for players,even in multiplayer.

File: content/blocks/crafting/adv-concrete-mixer.hjson [hunk 4]
```diff
@@ -0,0 +1,35 @@
+type: LiquidConverter
+name: Concrete mixer
+//you can balace stuffs if you want to
+description: "Mix sand and iron together to make concrete, require large amount of extra water"
+health: 1250
+size: 3
+requirements:
+[
+  refined-iron/50
```
Comment (BHLuotianyi) [inline content/blocks/crafting/adv-concrete-mixer.hjson:9]: The same thing to say like copper.In my opinion,producing large amounts of RF-iron should not be easy for players to do in the early game.RF-iron should be a material that players are always short of in the early and middle stage of game.

Comment (BHLuotianyi) [reply content/blocks/crafting/adv-concrete-mixer.hjson:9]: And building factories will certainly consumes lots of RF.

File: content/blocks/crafting/adv-uranium-refinery.hjson [hunk 5]
```diff
@@ -0,0 +1,34 @@
+type: GenericCrafter
+name: Advance Uranium Refinery
+description: "Refine depleted uranium in a large batch, making it more active again."
+health: 750
+size: 3
+requirements:
+[
+    steel/75
+    uranium-238/25
```
Comment (BHLuotianyi) [inline content/blocks/crafting/adv-uranium-refinery.hjson:9]: oof,why we need U-238 to build a factory?
_Probably prepared for executing workers that are not behaving well._

File: content/blocks/crafting/adv-uranium-refinery.hjson [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    solid-concrete/75
```
Comment (BHLuotianyi) [inline content/blocks/crafting/adv-uranium-refinery.hjson:10]: This should be used far more.The character of concrete is like copper in some ways,but it need to be produced.It's like a kind of _basic things_

File: content/blocks/crafting/adv-uranium-refinery.hjson [hunk 7]
```diff
@@ -0,0 +1,34 @@
+type: GenericCrafter
+name: Advance Uranium Refinery
+description: "Refine depleted uranium in a large batch, making it more active again."
+health: 750
+size: 3
+requirements:
+[
+    steel/75
```
Comment (BHLuotianyi) [inline content/blocks/crafting/adv-uranium-refinery.hjson:8]: _You're processing uramium,but not playing with muds._
Just increase the amount.The amout of steels produced per minute at the early stage of the game should be around 500 (4*full-used steelworks).

File: content/blocks/crafting/adv-uranium-refinery.hjson [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+    uranium-238/25
+    solid-concrete/75
+    refined-iron/35
```
Comment (BHLuotianyi) [inline content/blocks/crafting/adv-uranium-refinery.hjson:11]: _Bigger number,bigger happiness._

File: content/blocks/crafting/adv-uranium-refinery.hjson [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+    metaglass/250
+    graphite/110
+    silicon/75
```
Comment (BHLuotianyi) [inline content/blocks/crafting/adv-uranium-refinery.hjson:14]: Besides silicon,we should also add one middle-rank board.

File: content/blocks/crafting/adv-concrete-mixer.hjson (hunk 2 above)
Comment (BHLuotianyi) [reply content/blocks/crafting/adv-concrete-mixer.hjson:11]: Silicon and boards should be used together.

File: content/blocks/multi-block/crafting/steelworks-standard.hjson [hunk 10]
```diff
@@ -1,7 +1,7 @@
 {
   type: GenericSmelter
   name: Standard steelworks
-  description: Multi-block factory.Provides more steel.\n Needs to enter water.\n Needs to provide refined carbon and refined iron.\n This is the [red]first [orange]multi-block factory[]of THC Industry.Programed by[#66ccff]WinterUnderTheSnow[] on 2020.10.28.
+  description: "Multi-block factory.Provides more steel.\n Needs to enter water.\n Needs to provide refined carbon and refined iron.\n This is the [red]first [orange]multi-block factory[]of THC Industry.Programed by[#66ccff]WinterUnderTheSnow[] on 2020.10.28."
```
Comment (BHLuotianyi) [inline content/blocks/multi-block/crafting/steelworks-standard.hjson:4]: _SUPER SECRET : YOU CAN SAVE 2 BYTES OF DISK SPACE WHEN REMOVING AN `" "`_

**Review by BHLuotianyi — CHANGES_REQUESTED** [review]
Also,we should remove [this](https://github.com/THCworkshopCN/THC-Industry/blob/dev/content/blocks/crafting/steelworks-advanced.hjson)
A factory that uses quantum-state things must be a multi-block fatory.
Just try to build steelworks-standard in the sandbox.

File: content/blocks/crafting/adv-concrete-mixer.hjson (hunk 3 above)
Comment (BHLuotianyi) [reply content/blocks/crafting/adv-concrete-mixer.hjson:10]: _Like crafting two-three anvils immediately after you create a new world in Minecraft._

**Comment by Eclipse-04** [comment]
ah yes, ~~i can't play the game anymore~~

File: content/blocks/multi-block/crafting/steelworks-standard.hjson (hunk 10 above)
Comment (Eclipse-04) [reply content/blocks/multi-block/crafting/steelworks-standard.hjson:4]: emm... actually not placing "" sometime can cause problems
when using with [] or {} typo, i guess.

File: content/blocks/crafting/adv-concrete-mixer.hjson (hunk 1 above)
Comment (Eclipse-04) [reply content/blocks/crafting/adv-concrete-mixer.hjson:5]: ~~haha big number funi~~

**Comment by BHLuotianyi** [comment]
Let's merge this first.
I'll handle it.

**Pull request merged**